src/index.css

```diff
@@ -4,10 +4,10 @@ body {
     -apple-system,
     BlinkMacSystemFont,
     "Segoe UI",
-    "Roboto",
-    "Oxygen",
-    "Ubuntu",
-    "Cantarell",
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
     "Fira Sans",
     "Droid Sans",
     "Helvetica Neue",
```

Comment:
According to this requirement:

![image](https://user-images.githubusercontent.com/57421823/153107695-5ad0205f-cbbd-46fd-8fa8-de7361958307.png)

- [ ] These styles come straight from the CRA boilerplate.  Please, kindly remove them in order to clean up the repo, and to meet said requirement.